extensions/interactions/guppy.html

```diff
@@ -1,4 +1,10 @@
 <link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default.min.css">
+<script>
+  // Use the normal default browser eventlistener without passive set to false.
+  // TODO(jameesjohn): Use passive listeners when guppy supports it.
```

Comment:
```suggestion
  // TODO(#8833): Use passive listeners when guppy supports it.
```
we no longer use username TODOs